src/compiler/parser.ts

```diff
@@ -2143,16 +2166,31 @@ module ts {
             }
         }
 
+        function setContextParameters(explicitFlags: ParserContextFlags = ParserContextFlags.None, ambientFlags: ParserContextFlags = ParserContextFlags.None) {

```

Comment:
1) Does anyone not pass these parameters to setContextParameters?
2) We've found that default parameter values can actually be expensive.  Can these just be optional?